roles/dcos_bootstrap/tasks/main.yml

```diff
@@ -265,11 +265,28 @@
   command: "bash {{ install_win_file }}"
   args:
     chdir: "{{ download_path }}"
-    creates: "{{ download_path }}/genconf/serve/windows/prerequisites/dcos_install.ps1"
+    creates: "{{ download_path }}/genconf/serve/windows/prerequisites/prerequisites.zip"
```

Comment:
we use the file just as a marker to make module "command" idempotent

Reply:
i know how creates works. But the question is why is this an archive called prerequisites instead of dcos_install? What does prerequisites contain? Its pretty weird compared to linux install that genconf delivers something like prerequisites.

so again why is this going to be different to the linux install?

Reply:
dcos_install.ps1 fill up has been moved to gen , which means it moved to DC/OS Linux installer responsibility